broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -285,7 +273,19 @@ private synchronized RemotingCommand updateAndCreateTopic(ChannelHandlerContext
 
         this.brokerController.registerIncrementBrokerData(topicConfig, this.brokerController.getTopicConfigManager().getDataVersion());
 
-        return null;
+        response.setCode(ResponseCode.SUCCESS);
+        return response;
+    }
+
+    private boolean isSystemTopic(RemotingCommand response, String topic) {
+        if (this.brokerController.getTopicConfigManager().isSystemTopic(topic)) {
```

Comment:
It would be better to integrate the two methods: org.apache.rocketmq.common.MixAll#isSystemTopic

Reply:
The MixAll#isSystemTopic is used in NamespaceUtil, but NamespaceUtil is use in both broker and client. Some topic can be mixed like AUTO_CREATE_TOPIC_KEY_TOPIC, but some can't, such as the topic of broker' clusterName.
I try to use a new method in nameserver which offer the ability to judge if the topic will be created is system topic before this pr.But I found there always will be mistake when some broker register itself to nameserver after the judge request. 
So I just promise broker itself will not create topics which it thought is system topic.

Do u have any ideas to fix it?

Reply:
It is a good time to arrange the optimization for system topic or special inner topic globally in this pr.